Makefile

```diff
@@ -208,7 +213,10 @@ KIND = $(shell pwd)/bin/kind
 kind: ## Download kind locally if necessary.
 	$(call go-get-tool,$(KIND),sigs.k8s.io/kind@v0.11.1)
 
-ENVTEST = $(shell pwd)/bin/setup-envtest
 envtest: ## Download envtest-setup locally if necessary.
-	$(call go-get-tool,$(ENVTEST),sigs.k8s.io/controller-runtime/tools/setup-envtest@latest)
-	$(ENVTEST) use 1.21
+ifeq (, $(shell which setup-envtest))
+	go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
```

Comment:
I would rather pin a version instead of `latest`

Reply:
I thought of that too when Authorino made the update, then when the conversation happened, we realized it's a testing tool that won't change anything in the production code. Also the Operator SDK scaffolds it with `latest` tag